apromore/ApromoreCode
# AP-6981 support import export subprocesses with lanes

This PR adds support to include linked subprocesses with pools/lanes when exporting and import subprocess models with lanes.

When exporting a model with linked subprocesses included, If the linked subprocess model is a collaboration model (i.e. it contains pools), the collaboration model is converted into a single process diagram before being included in the exported bpmn. Lanesets in a subprocess are converted into participants in the subprocess model when the model is re-imported.

Other changes:
- Moved private methods in BPMNDocumentHelper below public methods.

To Test:
- Create a model with a subprocess
- Link the subprocess to a model with a pool
- Export the model with linked subprocesses included
- Re-import the model
- Check that the re-imported subprocess model looks the same as the original linked model.
